src/module.d.ts

```diff
@@ -851,7 +851,7 @@ declare namespace posthog {
 export class SentryIntegration implements Integration {
     constructor(posthog: posthog, organization?: string, projectId?: number, prefix?: string)
     name: string
-    setupOnce(addGlobalEventProcessor: (callback: EventProcessor) => void): void
+    setupOnce(addGlobalEventProcessor: (callback: EventProcessor) => void, getCurrentHub: () => Hub): void
```

Comment:
Hm, I don't think this is quite correct as we don't support `getCurrentHub` in https://github.com/PostHog/posthog-js/blob/7d5eefb21fb7634ae1a74d0fd099e730567aea89/src/posthog-core.js#L1493.

What's the intention with adding this and updating types to 7.2.0?

Reply:
@Twixes  in the latest release of [@sentry/types](https://www.npmjs.com/package/@sentry/types) 7.x.x, an additional parameter has been added in setupOnce function i.e. getCurrentHub. So due to this, I am not able to update this library to the latest version as I am using sentry postHog integration also. This update will to keep both sentry and postHog libraries to the latest versions.